etc/config/sway.defaults.properties

```diff
@@ -9,11 +9,11 @@ language.sway-ir.monaco=rust
 
 # Compiler settings
 compilers=&sway
-defaultCompiler=swayv0667
+defaultCompiler=swayv0689
 compilerType=sway-compiler
 
 # Define compiler setup
-group.sway.compilers=swayv0667
```

Comment:
I'm not quite sure how this was working before as `sway.default.properties` doesn't define a `swayv0667` or a `swayv0689` compiler...does this work for you? I'm happy to merge it as it's no worse than it was before but I think it's a little confusing and I don't think it would work for anyone.

Reply:
apologies for the delay in response @mattgodbolt , have just got back from some time off. Thanks for pointing this out! I'll address it in a follow up PR next time we update the compiler version. Cheers.